Marlin/temperature.cpp

```diff
@@ -394,6 +427,16 @@ uint8_t Temperature::soft_pwm_amount[HOTENDS],
         #endif
 
         temp_ms = ms + 2000UL;
+
+        #if WATCH_THE_BED || WATCH_HOTENDS
+          heated = heated || input > watch_temp_target;
```

Comment:
???

Reply:
I see I should have been more clear about what my pull request actually does.

Normally when a heater is on, thermal protection currently functions by ensuring that:
* The heater continually makes progress until the target temperature is reached
* Once the target temperature is reached, the temperature does not stray outside a specified range

Currently during M303, the only guarantee is that straying outside the min and max temperature range results in an error.  My pull request ensures that the above two conditions are also guaranteed during PID tuning.

Reply:
I can see what the code is doing. I just wanted to point out an errant line which I removed:
```cpp
heated = heated || input > watch_temp_target;
```